config.yaml

```diff
@@ -198,6 +234,41 @@ mappings:
           node_id: {type: text}
           label: {type: text}
 
+  "attribute_mapping:1":
+    global_mappings: [ws_subobject, ws_auth, ws_object]
+    properties:
+      attributes: {type: keyword}
+      attribute_ontology_ids: {type: keyword}
+      attribute_units: {type: keyword}
+      attribute_unit_ontology_ids: {type: keyword}
+      attribute_values: {type: keyword}
+      attribute_value_ontology_ids: {type: keyword}
+      instances: {type: object}
+      num_attributes: {type: integer}
+      num_instances: {type: integer}
+
+  "matrix:1":
+    global_mappings: [ws_auth, ws_object]
+    properties:
+      matrix_type: {type: keyword}
+      row_attributes: {type: keyword}
+      col_attributes: {type: keyword}
+      row_attribute_ontology_ids: {type: keyword}
+      col_attribute_ontology_ids: {type: keyword}
+      row_attribute_values: {type: keyword}
+      col_attribute_values: {type: keyword}
+      row_ids: {type: keyword}
+      col_ids: {type: keyword}
+      num_rows: {type: integer}
+      num_columns: {type: integer}
+      attributes: {type: keyword}
+
+  "genomeset:1":
+    global_mappings: [ws_auth, ws_object]
+    properties:
+      genomes: {type: object}
```

Comment:
maybe you can add a nested type mapping for this object

Reply:
👍